glide.yaml

```diff
@@ -13,9 +13,10 @@ import:
 - package: github.com/johanneswuerbach/nfsexports
 - package: github.com/mitchellh/go-ps
 - package: github.com/zchee/go-qcow2
-- package: github.com/zchee/libhyperkit
 - package: github.com/jceel/lib9p
   version: 7be3e0b914fbb2d2ecfe2984d02bacfd8e762eae
+- package: github.com/zchee/go-vmnet
+- package: github.com/moby/hyperkit
```

Comment:
We should put version v0.20170425 so that each iteration shouldn't get from master.

```
- package: github.com/moby/hyperkit
  version: v0.20170425
```